packages/server/src/schema/schema.ts

```diff
@@ -2,15 +2,9 @@ import { makeSchema } from 'nexus';
 import path from 'path';
 import { types } from './types';
 
-const parent = path.basename(path.join(__dirname, '..'));
-
 export const schema = makeSchema({
-  shouldGenerateArtifacts: parent !== 'dist',
+  shouldGenerateArtifacts: false,
```

Comment:
That is interesting, why is always `false`

Reply:
We weren't ever using it and so all it did was add to the build time. Now when watching,  restarts are near instantaneous. 

I think this is only useful if you are exporting the schema from the package so that it can be used elsewhere, but we're not doing that, and the graphql endpoint servers the schema anyway. Maybe we can use that to generate TypeScript definitions if we need to.